executable/Env.re

```diff
@@ -90,38 +140,19 @@ let run =
     multishell
       ? makeTemporarySymlink() : Lwt.return(Directories.globalCurrentVersion);
 
-  switch (shell) {
```

Comment:
When the log level was added to `Env.re` it was only added for bash and zsh, so with fnm 1.15.0 you can't change the log level with fish, since the log level isn't printed, and therefore isn't sourced.